cumulus/parachains/integration-tests/emulated/tests/assets/asset-hub-rococo/Cargo.toml

```diff
@@ -34,5 +34,6 @@ parachains-common = { path = "../../../../../../parachains/common" }
 cumulus-pallet-parachain-system = { path = "../../../../../../pallets/parachain-system", default-features = false }
 testnet-parachains-constants = { path = "../../../../../runtimes/constants", features = ["rococo"] }
 asset-hub-rococo-runtime = { path = "../../../../../runtimes/assets/asset-hub-rococo" }
+penpal-runtime = { path = "../../../../../runtimes/testing/penpal" }
```

Comment:
I am just thinking if it would be better to re-export stuff in `penpal-emulated-chain`, because we already import from there, e.g.:
```
pub use rococo_system_emulated_network::{
...
		penpal_emulated_chain::{
...
```

It is the same question for `use rococo_runtime::xcm_config::XcmConfig as RococoXcmConfig;`, this could be also re-exported from `rococo_system_emulated_network`, so we could also remove dependency `rococo-runtime` from here, maybe it could safe some compilation/build time? At least we won't import everything everywhere :)

Reply:
> maybe it could safe some compilation/build time

I do know the details of how the compiler works, but since `penpal-emulated-chain` already depends on `penpal-runtime`, I guess that adding the same dependency shouldn't affect compilation time? I might be wrong.

My concern with re-exporting is polluting `polkadot-sdk` with duplicates.

In any case, this is done everywhere for all tests, it is not specific to this PR. If you are still convinced about being a better option a new issue can be open.

Reply:
Actually... I just saw it is already re-exported :D

Reply:
> Actually... I just saw it is already re-exported :D

Only happens for Penpal. Not sure what's the best solution. In any case, as I said, if we decide to change it, as it affects other runtimes, I would prefer to do it in a different PR.